persist-cli/src/main/java/io/ballerina/persist/cmd/Migrate.java

```diff
@@ -478,6 +556,12 @@ private static List<String> findDifferences(Module previousModel, Module current
         convertListToQuery(QueryTypes.REMOVE_TABLE, removedEntities, queries);
         convertMapToQuery(QueryTypes.CHANGE_TYPE, changedFieldTypes, queries, addedEntities);
 
+        if (!differences.isEmpty()) {
+            errStream.println("\nDetailed list of differences: ");
```

Comment:
```suggestion
            errStream.println(System.lineSeparator() +"Detailed list of differences: ");
```
Update all other places